<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifyWindow</class>
 <widget class="QWidget" name="NotifyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>88</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>88</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>88</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>NodifyWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NotifyWindow
{
	font-family:微软雅黑;
}
QPushButton#pushButton_close{
	height:25px;
	width:25px;
	border-image: url(:/Images/close_mini_normal.png);
}
QPushButton#pushButton_close:hover{
	border-image: url(:/Images/close_mini_normal.png);
}
QPushButton#closeBtnBar:pressed{
	border-image: url(:/Images/close_mini_normal.png);
}

QProgressBar{
	min-height:1px;
	max-height:1px;
}
QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
		stop:0 #36CAD8,
		stop:0.7 #36CAD8,
		stop:1 #36CAD800);
}
QWidget#content_id,QWidget#content_msg_1,QWidget#content_msg_2
{
	font-size:12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <property name="minimumSize">
      <size>
       <width>278</width>
       <height>88</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>278</width>
       <height>88</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg" native="true">
        <property name="minimumSize">
         <size>
          <width>262</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>262</width>
          <height>72</height>
         </size>
        </property>
        <widget class="QWidget" name="contentpanel" native="true">
         <property name="geometry">
          <rect>
           <x>1</x>
           <y>1</y>
           <width>262</width>
           <height>72</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>262</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>262</width>
           <height>72</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>72</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>68</width>
              <height>72</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>13</number>
             </property>
             <property name="bottomMargin">
              <number>16</number>
             </property>
             <item>
              <widget class="QLabel" name="content_image">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>72</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>72</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>13</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>13</number>
             </property>
             <item>
              <widget class="QLabel" name="content_id">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>14</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>14</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="content_msg_1">
               <property name="minimumSize">
                <size>
                 <width>154</width>
                 <height>14</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>184</width>
                 <height>14</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="content_msg_2">
               <property name="minimumSize">
                <size>
                 <width>154</width>
                 <height>14</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>184</width>
                 <height>14</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="TitlebarWidget" name="Title_bar" native="true">
         <property name="geometry">
          <rect>
           <x>7</x>
           <y>0</y>
           <width>251</width>
           <height>27</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
